packages/block-editor/src/components/link-control/viewer-slot.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+
+const { Slot: ViewerSlot, Fill: ViewerFill } = createSlotFill(
+	'BlockEditorLinkControlViewer'
+);
+
+export { ViewerSlot, ViewerFill };
```

Comment:
Is this something we need to document and support?

Reply:
See above